otterdog/eclipse-ankaios.jsonnet

```
@@ -50,7 +50,8 @@ orgs.newOrg('automotive.ankaios', 'eclipse-ankaios') {
       environments: [
         orgs.newEnvironment('github-pages') {
           branch_policies+: [
-            "main"
+            "main",
+            "tag:v[0-9].[0-9].[0.9]"
```

The patch char set should also be written as a list:
```suggestion
            "tag:v[0-9].[0-9].[0-9]"
```
It seems like the otterdog documentation is wrong: https://otterdog.readthedocs.io/en/latest/reference/organization/repository/branch-or-tag/

I also think that versions like v1.11.2 will not be matched because of the single char `[0-9]` expression. I tried to find how to match them too, but the pattern is not a regex (otherwise the dot wouldn't work this was).

From [what I found out](https://docs.github.com/en/rest/deployments/branch-policies?apiVersion=2022-11-28#create-a-deployment-branch-policy), it's a ruby file globbing expression: https://ruby-doc.org/core-2.5.1/File.html#method-c-fnmatch
And it does not nicely allow to match one or more digits. We can also leave it to match only single digits version parts, but we should keep it in mind (or document it somewhere to prevent confusion some day).

---

@krucod3 Thanks for the investigation. Based on that I tend to change it to

```
tag:v*
```

These are the just the tags that are allowed to deploy. It is the same filter as used in our GHA:

https://github.com/eclipse-ankaios/ank-sdk-python/blob/fca274818ead9384bb0c5610c78d790c512a7b57/.github/workflows/release.yml#L6

---

Let's do it this way 👍

---

Done.